ethereum.md

```diff
@@ -523,26 +532,33 @@ The `"transactions"` key loads the transactions.
     rule load "transactions" : { TX } => load "transactions" : { #sortJSONList(TX) }
          requires notBool #isSorted(TX)
 
-    rule <k> load "transactions" : { "data" : TI , "gasLimit" : TG , "gasPrice" : TP , "nonce" : TN , "r" : TR , "s" : TS , "to" : TT , "v" : TW , "value" : TV , .JSONList } => . ... </k>
+    rule <k> load "transactions" : { "arguments" : [ ARGS ],  "data" : TI , "function" : FUNC, "gasLimit" : TG , "gasPrice" : TP , "nonce" : TN , "r" : TR , "s" : TS , "to" : TT , "v" : TW , "value" : TV , .JSONList } => . ... </k>
          <txOrder>   ... .List => ListItem(!ID) </txOrder>
          <txPending> ... .List => ListItem(!ID) </txPending>
          <messages>
            ( .Bag
           => <message>
-               <msgID>      !ID:Int                                 </msgID>
-               <txNonce>    #asUnsigned(#parseByteStack(TN))         </txNonce>
-               <txGasPrice> #asUnsigned(#parseByteStack(TP))         </txGasPrice>
-               <txGasLimit> #asUnsigned(#parseByteStack(TG))         </txGasLimit>
-               <sendto>     #asAccount(#parseByteStack(TT))          </sendto>
-               <value>      #asUnsigned(#parseByteStack(TV))         </value>
-               <v>          #asUnsigned(#parseByteStack(TW))         </v>
+               <msgID>      !ID:Int                              </msgID>
```

Comment:
Why is this showing up as a diff? I don't see any change in these lines.

Reply:
We added func and adjusted the whitespace.